src/script/page/MainContent/panels/preferences/AccountPreferences.tsx

```diff
@@ -129,22 +129,22 @@ const AccountPreferences: React.FC<AccountPreferencesProps> = ({
           alignItems: 'center',
           display: 'flex',
           flexDirection: 'column',
-          gap: 16,
-          paddingBottom: 32,
           width: 560,
         }}
       >
         <div
```

Comment:
maybe use the `H` tag here instead of `div`?

Reply:
yeah, might make sense